src/components/SingleTermView/MergePanel/ChipChanges.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react"
+import { Stack, Typography, Chip, Box } from "@mui/material"
+import MergeStatusWrapper from "./MergeStatusWrapper"
+import { compareArrays } from "../../../utils"
+import OpenInNewOutlinedIcon from "@mui/icons-material/OpenInNewOutlined"
+import { vars } from "../../../theme/variables"
+
+const { gray800 } = vars
+
+interface ChipChangesProps {
+  data: string[]
+  compareData: string[]
+  status: string
+  type: "existingID" | "synonym"
+}
+
+const ChipChanges: React.FC<ChipChangesProps> = ({ data, compareData, status, type }) => {
+  const differences = compareArrays(data, compareData)
+  
+  const title = type === "existingID" ? "Existing IDs" : "Synonyms"
```

Comment:
Re-using this for the className is fine, but we are implicitly saying that this component title will either be Existing IDs or Synonyms, can we simply display the title that we are passed from the parent component same as we do with the LabelChange component? Thanks